lib/uvobot/uvo_scraper.rb

```diff
@@ -8,7 +8,7 @@ class UvoScraper
     NEW_ISSUE_URL = "#{BULLETIN_URL}/vestnik-a-registre/vestnik-479.html".freeze
     IT_CONTRACTS_CODE = '48000000-8 72000000-5'.freeze
 
-    def initialize(parser = Uvobot::UvoParser, html_client = Curl)
+    def initialize(parser = Uvobot::UvoParser, html_client = HTTParty)
```

Comment:
Ked som zapol follow redirect tak to vycerpalo limit na redirects a padlo tak isto. Zatial som nepatral co pouzivaju na overovanie ci ide request z browsera.